src/MainWindow.vala

```diff
@@ -188,6 +196,15 @@ namespace PantheonCalculator {
             basic_grid.attach (button_ans, 2, 5, 1, 1);
             basic_grid.attach (button_calc, 3, 5, 1, 1);
 
+            var button_ms = new Button ("MS", _("Store value in memory"));
```

Comment:
The tooltip should indicate the stored value will overwrite any existing value (or maybe this button should be insensitive if the memory contains a value?) Perhaps "Set memory value" is clearer.  Maybe nice to actually include the value the memory will take.

As this is a destructive action maybe it should be styled as such?